src/api/customProvider.ts

```diff
@@ -239,8 +241,9 @@ function getAccountBasedTransactions(address: Address) {
     if (isRecipient) {
       // Recipient
       const amount = tx.recipients.reduce((a, b) => +a + +b.amount, 0);
+      const fixedAmount = amount.toFixed(ETH_FIXED_PRECISION);
```

Comment:
curious about the reason to lose precision there rather than just using `<instance>.toFixed()`?

Reply:
also, this seems ETH specific, will this code execute only for Ethereum-like accounts?

Reply:
Yes, it is ETH specific. The reason is the following one. Our products does not always export ETH amounts with the same precision as our custom external provider. Therefore, just using `<instance>.toFixed()` would lead to erroneous mismatches. For instance:

`0.123456789012 (our product) ≠ 0.1234567890 (custom provider)`

As we can see here, the data provided by the custom provider is not detailed enough. 

By using a common fixed-point notation for imported and actual operations, we avoid this kind of false-positive mismatches.

Reply:
gotcha. too bad that crypto apis is not providing real amounts